src/content/docs/usage/workspaces.mdx

```diff
@@ -423,7 +287,7 @@ Workspace MyWorkspace successfully deleted
 
 ### Non-Interactively
 
-Daytona allows you to delete one or more existing Workspaces non-interactively, allowing for quick and automated cleanup of your development environments without manual selection.
+Daytona provides an option to delete one or more existing Workspaces non-interactively, allowing for quick and automated cleanup of your development environments without manual selection.
```

Comment:
Can't comment below, but `forward` was changed. The `<PROJECT>` was removed. Update accordingly